app/src/main/res/values/strings.xml

```diff
@@ -668,5 +685,15 @@
 
     <string name="text_connected">Connected</string>
     <string name="text_disconnected">Disconnected</string>
+    <string name="voltage_channels">Voltage at Channels</string>
```

Comment:
@abhinavraj23 can you add these strings somewhere in the middle, may be next to `multimeter_knob_states`? Adding content to resource files at the end makes a lot of merge conflicts in almost every PR.